### DISCUSSION TITLE
在 Gatsby 中与 MDX 一起使用 KaTeX

### DISCUSSION THREAD

**Discussion by giscus** [kxxt/kxxt-website#84] [Thread]
# 在 Gatsby 中与 MDX 一起使用 KaTeX

我在给我的 Gatsby 站点(使用 MDX)添加 KaTeX 支持时踩了不少坑，因此我写了这篇文章，来简述一下在 2021 年，如何在 Gatsby 中优雅的使用 KaTeX。

https://kxxt.vercel.app/blog/using-katex-with-mdx-in-gatsby/

**Comment by flashshare** [comment]
Great, thanks!
